tools/score_test_generator/test_generator.cpp

```diff
@@ -213,12 +221,14 @@ void generateScore(
     // Prepare memory for generated scores
     unsigned long long totalSamples = nonces.size();
     scoreResults.resize(totalSamples);
+    scoreProcessingTimes.resize(totalSamples);
 
     bool writeFilePerSample = true;
 #pragma omp parallel for num_threads(threadsCount)

```

Comment:
does this require users to install anything else? if yes, I'd not recommend to use omp. We can use `std::thread` instead for native support

Reply:
This doesn't need to install anything, it is native support from  VisualStudio 2019, GCC 4.9, Clang3.9 ..